modules/net-vpc/routes.tf

```diff
@@ -17,6 +17,15 @@
 # tfdoc:file:description Route resources.
 
 locals {
+  _default_route = var.vpc_create && var.delete_default_routes_on_create ? {} : {
```

Comment:
This will create default_route also when `var.vpc_create` is set to false. Is this intended?

Reply:
should be fixed now. PTAL